### DISCUSSION TITLE
2022/02/13/Hexo%20+%20Butterfly%20%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E3%80%87%EF%BC%89%E5%89%8D%E8%A8%80/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#25] [General]
# 2022/02/13/Hexo%20+%20Butterfly%20%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E3%80%87%EF%BC%89%E5%89%8D%E8%A8%80/

恭喜你发现了宝藏！🎉🎉🎉  本系列是这个博客网站的搭建教程，个人的踩坑经历也会一并记录 因为需要面向小白，所以很多地方会讲的比较细碎 这个博客系统的运作流程是：在本地安装配置 Hexo框架 + Butterfly主题 ，然后根据 Markdown文档 生成静态文件，再推送至云端，公网上只需

https://nickxu.me/2022/02/13/Hexo%20+%20Butterfly%20%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E3%80%87%EF%BC%89%E5%89%8D%E8%A8%80/

**Comment by YXZ252426** [comment]
我准备跟着学长的教程搭建自己的个人博客了，希望成功
